kyaml/resid/gvk.go

```diff
@@ -163,7 +163,6 @@ var orderFirst = []string{
 	"StorageClass",
 	"CustomResourceDefinition",
 	"ServiceAccount",
-	"PodSecurityPolicy",
```

Comment:
I'd like to leave this one. Although strictly speaking we are free to remove it based on the version skew policy, in practice it costs us virtually nothing to leave it, and if we don't we may well get bug reports (e.g. about sort order changing) from Kustomize users who still run older versions in the interim.

Reply:
Reasonable to avoid bug reports for older versions.

Updated.